src/uct/Makefile.am

```diff
@@ -14,7 +14,7 @@ libuct_la_CPPFLAGS = $(BASE_CPPFLAGS) -I$(abs_top_srcdir)/src -I$(abs_top_buildd
 libuct_la_LDFLAGS  = -ldl -version-info $(SOVERSION)
 libuct_la_LIBADD   = $(LIBM) ../ucs/libucs.la
 libuct_ladir       = $(includedir)/uct
-libmlx5_ver        = \"`rpm -qf $(IBVERBS_DIR)/include/infiniband/mlx5_hw.h &>/dev/null && rpm -qf /usr/include/infiniband/mlx5_hw.h`\"
+libmlx5_ver        = \"`(rpm -qf $(IBVERBS_DIR)/include/infiniband/mlx5_hw.h 2>/dev/null || rpm -qf /usr/include/infiniband/mlx5_hw.h 2>/dev/null) | head -1`\"
```

Comment:
it will break debian/ubuntu distros which have no rpm but other pkg manager

Reply:
how will it break? errors are redirected to devnull

Reply:
maybe logic should be as following:
* extract mofed/libmlx5 version with grep from /bin/ofed_info:
```
$head -10 /bin/ofed_info
#!/bin/bash
if [ "$1" == "-s" ]; then echo MLNX_OFED_LINUX-3.3-1.0.0.0:; exit; fi
cat << EOF
MLNX_OFED_LINUX-3.3-1.0.0.0 (OFED-3.3-1.0.0):

...
```

Reply:
```
$head  -2 /bin/ofed_info|grep MLNX|awk '{print $9}'| sed -e 's/:;//g'
MLNX_OFED_LINUX-3.3-1.0.0.0
```

Reply:
on system w/o rpm, rpm not found, empty command redirected to "|head -1"

Reply:
@miked-mellanox not sure we want to assume OFED existence
worst case, on system w/o rpm, libmlx5_ver will be empty, so the user error message will not be very useful

Reply:
mlx5_hw.h exists only in mofed